client/src/app/components/page/task-detail/task-detail.component.html

```diff
@@ -1,27 +1,35 @@
-<div class="card mt-4 container flex-row">
+<div class="mt-4 container border rounded-lg">
     <div class="row">
         <div class="colCard col-sm">
             <form [formGroup]="taskForm" (submit)="onSubmit()">
                 <h5 class="font-weight-light text-center text-lg-left mt-2 mb-2">Title</h5>
```

Comment:
For every input element use `label` for better SEO and accessibility (like screen readers etc). You can use "Form Group" or "Form Grid" https://getbootstrap.com/docs/4.5/components/forms/#form-groups. Please do it for every form element.

Reply:
Changed to label and added form groups